test/static_test/tensor/fixed_size_tuple.cpp

```diff
@@ -39,6 +39,12 @@ static_assert(h2::all_of(test_tuple,
 static_assert(h2::map(test_tuple, [](TestFixedSizeTuple::type x) { return x; })
                   == TestFixedSizeTuple{},
               "Map for empty tuples is wrong");
+static_assert(h2::map_index(test_tuple,
+                            [](TestFixedSizeTuple::size_type x) {
+                              return test_tuple[x];
+                            })
+                  == TestFixedSizeTuple{},
+              "Map index for empty tuples is wrong");
```

Comment:
I'm not sure these strings are adding much value. Might just want to remove them (ok in C++17+).